docs/data/capture/_index.md

```diff
@@ -367,6 +367,18 @@ The following example captures data from the `ReadImage` method of a camera:
 
 {{% /expand%}}
 
+## View captured data
+
+To view captured data for a {{< glossary_tooltip term_id="part" text="machine part" >}}, click on the menu in the top right of its card and select **View captured data**.
+
+{{<imgproc src="/data/part-menu.png" resize="300x" declaredimensions=true alt="Machine menu with the options Rename, Restart part, View captured data, View setup instructions, View history, View debug configuration, and Delete machine">}}
+
+To view captured data for a component or service, click on the menu in the top right of its card and select **View captured data**.
```

Comment:
just adding in case you didn't know that the same menu appears from the left pane list of resources when you hover. Don't think you need to call that out here too

Reply:
I've changed the wording slightly to account for that thanks!